ui/pom.xml

```diff
@@ -31,6 +31,13 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.jboss.windup.graph</groupId>
+            <artifactId>windup-graph-impl</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency> 
+
+        <dependency>
```

Comment:
yeah depending on impl here is bad. it actually won't work in future versions of furnace.